v2.0/releases/index.md

```diff
@@ -6,6 +6,40 @@ The following table shows component versioning for Calico  **{{ page.version }}*
 
 Use the version selector at the top-right of this page to view a different release.
 
+
+## v2.0.1 - Release Notes
+
+This is a bug fix release, the major fixes and enhancements include:
+- Fix: some policies not working because of / in the labels.
+- Fix that ipv4_nat was not working (for OpenStack floating IPs)
+- Fix logic in static link test. Move inside container.
+- Fix configure_ipip_device() to pass IPAddress to set_interface_ips().
```

Comment:
Similarly here; in the Felix notes I changed this to "Fix for projectcalico/calicoctl#1419."